react-class-components/src/index.jsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+class CustomButton extends React.Component {
+  render(props) {
+    return <button>this.props.text</button>;
```

Comment:
Your buttons do not look right. Use a JavaScript expression, in `{}` curly braces.